src/commands/test.js

```diff
@@ -13,8 +13,7 @@ const test = (context) => {
     extraEnv.DETAILED_LOG_TO_STDOUT = 'true';
   }
 
-  if (process.version !== LAMBDA_VERSION) {
-    context.line(`You're running tests on Node ${process.version}, but Zapier runs your code on ${LAMBDA_VERSION}. The version numbers must match. See https://zapier.github.io/zapier-platform-cli/index.html#requirements for more info.`);
+  if (!utils.correctVersion()) {
```

Comment:
I think how you designed it now is :100:, just a name to help communicate the intent of the function.

My question on context was that the new function assumes context exists and does `context.line`, so if we don't pass it, won't that blow up?